starter-code/index.html

```diff
@@ -4,39 +4,68 @@
   <meta charset="UTF-8">
   <title>Spotify</title>
 </head>
+<link rel="stylesheet" href="./css/main.css">
 <body>
-Premium Discover Help Download
+<header>
+  <navbar>
+      <img src = "./images/spotify-logo.png">
+      <ul> 
+          <li><a href = '#'> Premium</a></li>
+          <li><a href = '#'> Discover</a></li>
+          <li><a href = '#'> Help</a></li> 
+          <li><a href = '#'> Download</a></li>
+      </ul>
+  </navbar>
+</header>
 
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
+<!-- <hr> -->
+<div class = "large-image-with-background-text">
+  <p>
+    <span class = "top-line">Music for everyone.</span>
+    
+    <br><br><br>
+    
+    <span class = "bottom-line">Spotify is now free on mobile, tablet and computer.
+    Listen to the right music, wherever you are.</span>
+  </p>
+</div>
 
-What’s on Spotify?
+<div class ="whats-on-spotify">
+  <span class = "title"><p>What’s on Spotify?</p></span>
 
-Millions of Songs
-There are millions of
-songs on Spotify
 
-HD Music
-Listen to music as if you
-were listening live
+   <br> 
+   <span class = "mid-logos"><img src = "./images/music-icon.png" alt="music"></span>
+   
+    <span class = "green-text">Millions of Songs
+    There are millions of
+    songs on Spotify
 
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
+    <span class = "mid-logos"><img src = "./images/high-quality-icon.png" alt="music"></span>
+    <span class = "green-text">HD Music</span>
+    Listen to music as if you
+    were listening live
 
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+    <span class = "mid-logos"><img src = "./images/devices-icon.png" alt="music"></span>
+    <span class = "green-text">Stream Everywhere</span>
+    Stream music on your
+    smartphone, tablet or
+    computer
+</div>
```

Comment:
Please start getting into the habit of properly spacing/tabbing ALL of your divs, because all of your divs are on the same left side as the body tag, so now it is hard to follow where starting and closing tags start and end.